lib/index.js

```diff
@@ -149,7 +151,7 @@ function getOptions(options) {
 
   options.error = function(err) {
     if (error) {
-      error(util._extend(new Error(), JSON.parse(err)));
+      error.call(options.context, util._extend(new Error(), JSON.parse(err)));
```

Comment:
I don't think that would be a good idea. It would paint us into a bit of a corner if we need to surface any additional environment information from libsass or node-sass to callbacks.